src/App.svelte

```diff
@@ -225,11 +224,11 @@
     <Route path="/volume-df" primary={false}>
       <DataPortal />
     </Route>
-    <Route path="/veocean" primary={false}>
+    <Route path="/passive-df" primary={false}>
       <VeOceanPortal />
     </Route>
-    <Route path="/delegate" primary={false}>
-      <DelegationPortal />
```

Comment:
Hi Norbert, please do not delete routes.
Simply update the route and push the user to the right place.